docs_src/design/adr/0026-Global-Configuration.md

```diff
@@ -0,0 +1,70 @@
+# Global Common Configuration ADR
+
+## Status
+
+**Draft**
+
+This is a preliminary document outlining alternative design options for implementation of the Global Common Configuration.
+
+## Submitters
+
+- Jim White with community input
+
+## Change Log
+
+- N/A
+
+## Referenced Use Case(s)
+
+- [Global Common Configuration UCR – still to be approved]( https://github.com/edgexfoundry/edgex-docs/pull/892)
+
+## Context
+
+Per the UCR, there is a need to have global configuration so that common configuration is not repeated in each service’s configuration.  In other words, there is a goal to reduce redundancy in service configuration and allow for better management of the services configuration across all EdgeX services.
+
+As there have been multiple design / implementation alternatives suggested in planning meetings to date, this draft ADR is meant to serve as a landing area for all proposed implementation options.
+
+Options will be reduced to a single, selected design going forward and will serve as the basis of this ADR.  All other options will be relegated to the considerations section at that time.
+
+## Proposed Design(s)
```

Comment:
For reference here is a flow chart of the existing implementation for loading configuration at start-up.
![image](https://user-images.githubusercontent.com/44779287/204324398-34478ffd-22c1-48f4-b2ce-2aaa0ddb10d3.png)

Reply:
IMO, the SecretStore configuration should be removed from the TOML and have default value that can be overridden with environment variables.  We prefer this config to not be in the Config Provider, but it is currently present due to how the config loading is implemented.

Reply:
Here is proposed 3.0 flowchart using @farshidtz p
![image](https://user-images.githubusercontent.com/44779287/204335925-5dbc68fc-4c51-41a8-8654-ced86192e826.png)
roposal of not pushing all of service's config into Config Provider.

Reply:
> Here is proposed 3.0 flowchart using @farshidtz proposal of not pushing all of service's config into Config Provider.

Thanks for the flowchart. For clarity, that is for what I suggested at https://github.com/edgexfoundry/edgex-docs/pull/892#discussion_r1021320773, assuming that we have the existing Config Provider and add the separate global config provider.

What @jpwhitemn has attributed to be in this ADR is about not having a separate global config provider and simply seeding the Config Provider for that matter.